lib/engine/tile.rb

```diff
@@ -467,9 +467,10 @@ def separate_parts
       end
 
       @nodes = @paths.flat_map(&:nodes).uniq
-      @branches = @paths.flat_map(&:branches).uniq
       @stops = @paths.flat_map(&:stops).uniq
       @edges = @paths.flat_map(&:edges).uniq
+
+      @edges.each { |e| e.tile = self }
```

Comment:
did this not work previously?

Reply:
why is this needed?

Reply:
No. I thought tile.separate_parts would have done this too in lines 462-467, but it wasn't working and truthfully I wasn't sure what that code was supposed to do.